app/lib/theme/widgets/MoneyWidget.tsx

```diff
@@ -29,7 +29,7 @@ export const MoneyWidget: React.FC<WidgetProps> = ({
         className="money"
         decimalScale={2}
         defaultValue={(schema as any).defaultValue}
-        value={value}
+        value={value || ""}
```

Comment:
If it's an empty string, it does show up in `newFormData` like this: 
![Screenshot from 2022-04-19 10-40-18](https://user-images.githubusercontent.com/77306764/164063334-019ddeed-8129-4715-af9b-a9d7cd6926cc.png)